src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -88,12 +107,66 @@ public String getAdditionalSourceFileLocation() {
         return Arrays.asList(objectFile.toAbsolutePath().toString());
     }
 
+    @Override
+    public boolean link(ProcessPaths paths, ProjectConfiguration projectConfiguration) throws IOException, InterruptedException {
+        boolean result = super.link(paths, projectConfiguration);
+
+        if (result) {
+            createInfoPlist(paths, projectConfiguration);
+
+            if (! isSimulator()) {
+                CodeSigning codeSigning = new CodeSigning(paths, projectConfiguration);
+                if (! codeSigning.signApp()) {
+                    Logger.logSevere("Error signing the app");
+                }
+            }
+        }
+        return result;
+    }
+
     @Override
     public String getCompiler() {
         return "clang";
     }
 
+    @Override
+    String getAppPath(String appName) {
```

Comment:
add doc (to the interface) that tells that this will create the app path if it doesn't exist.
Probably better to rename this to "createAppPath" then?

Reply:
Done.
We don't have javadoc in any of the methods added to `AbstractTargetConfiguration`. I say we do this in a follow-up PR once Substrate is more stable?